pyth/plugins/rtf15/reader.py

```diff
@@ -542,25 +559,27 @@ def handle_ansi_escape(self, code):
 
     def handle_control_symbol(self, symbol):
         # Ignore ~, -, and _, since they are optional crap.
-        if symbol in '\\{}':
-            self.content.append(unicode(symbol))
+        assert len(symbol) == 1
```

Comment:
That was a "partial program understanding notice".
Just take it out if it makes you nervous.
And while you're at it: Take out line 564 as well; I overlooked it when taking out its various cousins.